Dockerfile-3_11

```diff
@@ -24,12 +24,20 @@ ADD https://github.com/datastax/metric-collector-for-apache-cassandra/releases/d
 RUN mkdir /opt/mcac-agent && tar zxvf /opt/mcac-agent.tar.gz -C /opt/mcac-agent --strip-components 1 && rm /opt/mcac-agent.tar.gz
 
 # backwards compat with upstream ENTRYPOINT
-COPY scripts/docker-entrypoint.sh /usr/local/bin/
+COPY scripts/no-gosu-docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
```

Comment:
I added an additional version of the entrypoint script so that the other Dockerfiles will not be affected by this change.

Reply:
Err, so this Dockerfile probably needs to be deleted. If you look at our build for 3.11:

https://github.com/datastax/management-api-for-apache-cassandra/blob/master/.github/workflows/docker-release.yaml#L64-L69

You'll see we actually use the Dockerfile-oss now. For your purposes, you should find the change essentially the same.

Reply:
Actually, if you could verify this Dockerfile isn't still referenced somewhere and delete it, I'd appreciate it. I should have gotten rid of it when I made the arm changes to prevent confusion.